wear/src/main/java/io/homeassistant/companion/android/home/HomeActivity.kt

```diff
@@ -61,6 +61,28 @@ class HomeActivity : ComponentActivity(), HomeView {
         SensorWorker.start(this)
 
         mainViewModel.initAllSensors()
+
+        if (mainViewModel.loadingState.value == MainViewModel.LoadingState.READY) {
+            lifecycleScope.launch {
+                launch {
+                    mainViewModel.entityRegistry = presenter.getEntityRegistry()
+                }
+                launch {
+                    mainViewModel.deviceRegistry = presenter.getDeviceRegistry()
+                }
+                launch {
+                    mainViewModel.areaRegistry = presenter.getAreaRegistry()?.also {
+                        mainViewModel.areas.addAll(it)
+                    }
+                }
+                launch {
```

Comment:
This should wait until the entity registry update has been completed instead of also using launch (= executed in parallel)